src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ServerIOExceptionFromReadIT.java

```diff
@@ -105,4 +105,26 @@ public void shouldNotResetWhenProcessingEndAfterIOExceptionFromRead() throws Exc
         }
     }
 
+    @Test
+    @Specification({
+        "${route}/server/controller",
+        "${server}/server.sent.reset.then.end/server"
+    })
+    public void shouldNotResetWhenProcessingEndAfterIOExceptionFromRead() throws Exception
```

Comment:
Same applies as for the corresponding client test method (this message go into ServerResetAndAbortIT).

Reply:
Ok, done.